.changeset/dull-maps-draw.md

```diff
@@ -0,0 +1,8 @@
+---
+'@ts-rest/nest': minor
+'@ts-rest/core': patch
+---
+
+Rename some Nest functions and types, and deprecate old names
+
+Fix Nest deprecation warning when passing Zod error to HttpException (#122)
```

Comment:
Should we add the removal of the exports here? 

Or does that go somewhere else?

Reply:
Yup, here sounds about right in this case